PORTAL/jobs/_pyperformance.py

```diff
@@ -2470,7 +2858,7 @@ def add_from_results(self, results, *,
             logger.info(f'...as {resfile.relfile}...')
             #copied = suite_results.copy_to(resfile, self._root)
             copied.append(
-                suite_results.copy_to(resfile, self._root)
+                suite_results.copy_to(resfile.filename, self._root)
```

Comment:
Something about this seems off.  I'm not sure what.  Let's just keep the original.